### PR TITLE
etcd-tester: fix tester for 5-node cluster

### PR DESCRIPTION
**Pull request by gyuho**
1. fix failure case counting
2. match ErrClientConnClosing in stresser
3. longer timeout for set-health-key
4. fixed range for range/delete stresser

### CODE REVIEW

File: tools/functional-tester/etcd-tester/stresser.go [hunk 1]
```diff
@@ -104,21 +105,22 @@ func newStressRangeInterval(kvc pb.KVClient, keySuffixRange, keyRangeLimit int)
...
+			Key: []byte(fmt.Sprintf("foo%016x", rand.Intn(keySuffixRange))),
 		}, grpc.FailFast(false))
 		return err
 	}
 }
 
 func newStressDeleteInterval(kvc pb.KVClient, keySuffixRange, keyRangeLimit int) stressFunc {
 	return func(ctx context.Context) error {
+		mid := keySuffixRange / 2
 		resp, _ := kvc.Range(ctx, &pb.RangeRequest{
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/stresser.go]: why do we still need this range ?

Comment (gyuho) [inline tools/functional-tester/etcd-tester/stresser.go]: Don't think we need it anymore. Will remove. Thanks.

File: tools/functional-tester/etcd-tester/stresser.go [hunk 2]
```diff
@@ -84,17 +84,18 @@ func newStressPut(kvc pb.KVClient, keySuffixRange, keySize int) stressFunc {
...
-			Key: []byte(fmt.Sprintf("foo%d", rand.Intn(keySuffixRange))),
+			Key: []byte(fmt.Sprintf("foo%016x", rand.Intn(keySuffixRange))),
 		}, grpc.FailFast(false))
 		return err
 	}
 }
 
 func newStressRangeInterval(kvc pb.KVClient, keySuffixRange, keyRangeLimit int) stressFunc {
 	return func(ctx context.Context) error {
+		mid := keySuffixRange / 2
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/stresser.go]: probably just remove this. and do start = intN(keySuffixRange), end = start + 500.

Comment (xiang90) [inline tools/functional-tester/etcd-tester/stresser.go]: same for deleterange.

**Comment by xiang90** [comment]
lgtm

File: tools/functional-tester/etcd-tester/stresser.go [hunk 3]
```diff
@@ -192,7 +184,7 @@ func (s *stresser) Stress() error {
 		{weight: 0.07, f: newStressRange(kvc, s.keySuffixRange)},
 		{weight: 0.07, f: newStressRangeInterval(kvc, s.keySuffixRange, s.keyRangeLimit)},
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/stresser.go]: remove keyRangeLimit?

**Pull request merged**